mypy/typeanal.py

```diff
@@ -148,6 +148,14 @@ def visit_unbound_type(self, t: UnboundType) -> Type:
                 items = self.anal_array(t.args)
                 item = items[0]
                 return TypeType(item, line=t.line)
+            elif fullname == 'typing.ClassVar':
+                if len(t.args) == 0:
+                    return AnyType(line=t.line)
+                if len(t.args) != 1:
+                    self.fail('ClassVar[...] must have exactly one type argument', t)
```

Comment:
The error message could be misleading, change it to "... at most one type argument"